package.json

```diff
@@ -34,7 +34,8 @@
   "scripts": {
     "cover": "nyc --reporter=html --reporter=text npm test",
     "lint": "standard \"**/*.js\" \"**/*.html\" \"**/*.md\"",
-    "test": "cross-env NODE_ENV=test mocha $npm_package_options_mocha",
+    "test": "cross-env NODE_ENV=test mocha test/unit $npm_package_options_mocha",
```

Comment:
i am directly using `nyc` so with test, we can also see coverage.

https://github.com/liquality/chainabstractionlayer/commit/103fbe94d6cdbf0d61533b04ad8f25ef370aa77c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R37

Reply:
The purpose of these tests are only to run locally so it's kind of strange to reflect the code coverage in it. It wouldn't be relevant to someone running it.
Perhaps we need an overall `test` script that runs `nyc, npm run test:unit, npm run test:swap` ?